src/cascadia/UnitTests_TerminalCore/ConptyRoundtripTests.cpp

```diff
@@ -174,6 +180,14 @@ class TerminalCoreUnitTests::ConptyRoundtripTests final
 
     TEST_METHOD(TestResizeHeight);
 
+    TEST_METHOD(OutputWrappedLinesAtTopOfBuffer);
+    TEST_METHOD(OutputWrappedLinesAtBottomOfBuffer);
+    TEST_METHOD(ScrollWithChangesInMiddle);
+    TEST_METHOD(DontWrapMoveCursorInSingleFrame);
+    TEST_METHOD(ClearHostTrickeryTest);
+    TEST_METHOD(OverstrikeAtBottomOfBuffer);
+    TEST_METHOD(MarginsWithStatusLine);

```

Comment:
OK so I was mildly concerned with the complexity of the trickery going on in the `XtermEngine` prior to looking at this. But if you wrote this many tests around it, I'm now less concerned.